### PR TITLE
Point users to latest released code

### PR DESCRIPTION
**Pull request by rhyshort**
## What
Point users to the latest code

## Why
For the sample application some users download the current
version on master, this at times relies on code that has yet
to be released. To combat this a small note (in bold) has
been added to the README pointing to the latest release page,
where the code can be downloaded.

## Issues 

#295

### CODE REVIEW

File: sample/todo-sync/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Example application: Todo Sync
 
+** The latest released sample version can be downloaded from the
+[GitHub releases page][latest].**
+
+[latest]: https://github.com/cloudant/releases/latest
```
Comment (tomblench) [inline sample/todo-sync/README.md]: I think you mean https://github.com/cloudant/sync-android/releases/latest

File: sample/todo-sync/README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Example application: Todo Sync
 
+** The latest released sample version can be downloaded from the
```
Comment (tomblench) [inline sample/todo-sync/README.md]: Are the `**` meant to be some sort of markup? Should this be bold/italic etc?

Comment (rhyshort) [inline sample/todo-sync/README.md]: Yeah it should have made it bold,  I need to remove the space.

File: sample/todo-sync/README.md (hunk 1 above)
Comment (rhyshort) [inline sample/todo-sync/README.md]: yeah, fixed in f0c372e

File: sample/todo-sync/README.md (hunk 2 above)
Comment (rhyshort) [inline sample/todo-sync/README.md]: Fixed in f0c372e

**Pull request merged**